app/components/site-head.js

```diff
@@ -16,14 +16,14 @@ export default Component.extend(
 		smartBanner: service(),
 
 		classNames: ['site-head-container'],
-		classNameBindings: ['themeBar'],
+		classNameBindings: ['themeBar', 'partnerSlot:site-head__has-partner-slot'],
```

Comment:
I'd name the class `has-partner-slot` without `site-head__` prefiex. This is how we do it in other places.